### PR TITLE
Some docs and setup tweaks while familiarizing myself with the code.

### PR DESCRIPTION
**Pull request by rfk**
I was poking around making sure I understood the moving parts here, and figured I'd collect a bunch of nits into a PR as part of that.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,7 +15,6 @@ build
 captures
 .externalNativeBuild
 .lastAutoPublishContentsHash
-# Not sure if we should ignore those, but we don't check them
-# into Application Services
```
Comment (rfk) [inline .gitignore:19]: AFAICT we do actually commit these in both appservices and a-c, and they're necessary in order for the `./gradlew` wrapper script to work.

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Cross Platform Rapid Experiments "Nimbus" SDK
 * **[Issue Tracker / Epic](https://jira.mozilla.com/browse/SYNC-1528)**
 * **[Project Plan Page](https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=126619091)**
 * [Bucketing Technical Documentation](https://docs.google.com/document/d/1WAForAUIchVPaiZFCJO3hNQHY_7KZAjddfscTM_Lx0Y/edit#)
-* [Nimbus Mana Page](https://mana.mozilla.org/wiki/display/PM/Project+Nimbus)
```
Comment (rfk) [inline README.md:9]: This page looked out of date, so I updated it with a more recent one.

File: android/README.md [hunk 3]
```diff
@@ -3,9 +3,9 @@ This directory contains the build script that compiles the Nimbus SDK and genera
 bindings
 
 ## Prerequisites
-- `uniffi-bindgen`: The library uses `uniffi` to generate the bindings, you will need to install
```
Comment (rfk) [inline android/README.md:6]: `uniffi` has now been published to crates.io 🎉

File: android/src/main/README.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Placeholder directory for generated Kotlin bindings
+
+Expecting to see some `.kt` source files in this directory?
+Don't worry, you're in the right place! It's just that all our
+Kotlin files are generated automatically as part of the gradle
+build process, using `uniffi-bindgen`.
```
Comment (rfk) [inline android/src/main/README.md:6]: The empty directory here briefly alarmed me until I remembered that all the hand-written Kotlin was going to be in a-c 😆

**Review by travis79 — APPROVED** [review]
Thanks for jumping in here Ryan, and thanks for cleaning things up a bit!

**Review by vladikoff — APPROVED** [review]
🎉

**Pull request merged**